package.json

```diff
@@ -27,7 +27,7 @@
     "i18n:extract": "yarn workspaces run i18n:extract",
     "i18n:compile": "yarn workspaces run i18n:compile",
     "lint": "yarn workspaces run lint",
-    "test": "yarn workspace @openfun/verna run test"
+    "test": "yarn workspace @openfun/verna run test && yarn workspace verna-examples-poll run test"
```

Comment:
Unfortunately you cannot write this command like. In ci we use the flag `--runInBand` but with this command this flag will be applied only to the test suite of `verna-examples-poll`.

A workaround would to be tu use `yarn workspaces run test` then add a test script in each workspace. For those who haven't test suite, you can add use `jest --passWithNoTest`.